docs-2.0/7.data-security/2.backup-restore/4.br-restore-data.md

```diff
@@ -16,67 +16,105 @@ To restore data with the BR, do a check of these:
-
-  !!! caution
+- No application is connected to the target Nebula Graph cluster.
 
-      Run `ln -s /<ossutil_tool_installation_path>/<ossutil64 or ossutil> /usr/local/bin/ossutil` to make the `ossutil` command effective.
+- Make sure that the target and the source Nebula Graph clusters have the same topology, which means that they have exactly the same number of hosts. The number of data folders for each host is consistently  distributed.
 
 ## Procedures
 
 1. Users can use the following command to list the existing backup information:
 
-   ```bash
-   $ ./bin/br show --storage <ip_address>
-   ```
-   For example, use the following command to list the backup information in the local `/home/test/backup` path.
-   ```bash
-   $ ./bin/br show --storage "local:///home/test/backup"
-   +----------------------------+---------------------+------------------+-------------+---------------+
-   |            NAME            |     CREATE TIME     |      SPACES      | FULL BACKUP | SPECIFY SPACE |
-   +----------------------------+---------------------+------------------+-------------+---------------+
-   | BACKUP_2021_07_29_06_51_09 | 2021-07-29 06:51:09 | basketballplayer | true        | true          |
-   | BACKUP_2021_07_29_07_55_08 | 2021-07-29 07:55:09 | basketballplayer | true        | true          |
-   +----------------------------+---------------------+------------------+-------------+---------------+
-   ```
+  ```bash
+  $ ./bin/br show --storage <ip_address>
+  ```
+  For example, use the following command to list the backup information in the local `/home/nebula/backup` path.
+  ```bash
+  $ ./bin/br show --storage "local:///home/nebula/backup"
+  +----------------------------+---------------------+------------------------+-------------+------------+
+  |            NAME            |     CREATE TIME     |         SPACES         | FULL BACKUP | ALL SPACES |
+  +----------------------------+---------------------+------------------------+-------------+------------+
+  | BACKUP_2022_02_10_07_40_41 | 2022-02-10 07:40:41 | basketballplayer       | true        | true       |
+  | BACKUP_2022_02_11_08_26_43 | 2022-02-11 08:26:47 | basketballplayer,foesa | true        | true       |
+  +----------------------------+---------------------+------------------------+-------------+------------+
+  ```
+
+  Or, you can use the following command to list the backup information in S3 URL `s3://127.0.0.1:9000/br-test/backup`.
+  ```bash
+  $ ./bin/br show --s3.endpoint "http://127.0.0.1:9000" --storage="s3://br-test/backup/" --s3.access_key=minioadmin --s3.secret_key=minioadmin --s3.region=default
+  ```
+
+  | Parameter    | Data type | Required | Default value | Description                                                                 |
```

Comment:
![image](https://user-images.githubusercontent.com/78209557/154184135-b6921c7d-d1f8-40dd-a9e2-531853c879d4.png)
seems the English content is different from the Chinese.